app/bin/tools/remove_package.dart

```diff
@@ -43,7 +41,7 @@ Future main(List<String> arguments) async {
       await listPackage(package);
     } else if (command == 'remove') {
       if (version == null) {
-        await removePackage(package);
+        await _removePackage(package);
```

Comment:
This changes semantics by creating a `ModeratedPackage` entity... which is probably fine :D